src/components/Token/Hero/index.tsx

```diff
@@ -0,0 +1,91 @@
+import css from './styles.module.css'
+import { motion } from 'framer-motion'
+import { Button, Typography } from '@mui/material'
+import OrbitRingsList from '@/content/token-hero-rings.json'
+import OrbitPath from '@/components/Token/OrbitPath'
+import Floater from '@/components/Token/Floaters'
+import PulsingLogo from '@/components/Token/PulsingLogo'
+
+export default function Hero() {
+  return (
+    <div className={css.heroSpacer}>
+      {/* Token Backgroud Gradients */}
+      <div className={css.tokenCircle1}></div>
+      <div className={css.tokenCircle2}></div>
+      <div className={css.tokenCircle3}></div>
+
+      <div className={css.tokenContainer}>
+        {/*SAFE Token Logo */}
+        <motion.div
+          animate={{ y: ['0%', '-5%', '0%'], rotateY: [0, -35, 0] }}
+          transition={{
+            duration: 6,
+            ease: 'easeInOut',
+            repeat: Infinity,
+            repeatType: 'loop',
+          }}
+          className={css.tokenImage}
+        >
+          <img width={180} src="/images/Token/token.png" alt="Safe Token" />
+        </motion.div>
+
+        {/* Token Hero Content */}
+        <div
+          className={css.container}
+          style={{
+            position: 'absolute',
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            top: '220px',
+          }}
+        >
+          <Typography className={css.label} variant="h4">
+            SAFE Token
+          </Typography>
+          <Typography variant="h2" className={css.title}>
+            Participate in the
+            <span> Ownership Revolution</span>
+          </Typography>
+          <Button className={css.button} variant="contained" size="large">
```

Comment:
```suggestion
          <Button className={css.button} variant="contained" size="large" href={WALLET_LINK}>
```

remember to import the const